### PR TITLE
Remove list of available APIs from README

### LINKED ISSUE #297: Readme contains incorrect documentation for deleteSource method
The readme documentation specifies the deleteSource as having the following signature:

`deleteSource(customerId, {source: 'source_token'})`

When the method is actually supposed to be the following:

`deleteSource(customerId, source_token)`

This took me a while to diagnose, as these methods are sort of dynamic in the source code, and you can't quite understand what is going on without being an expert at the implementation.

### LINKED ISSUE #298: Readme functions out of date.  Functions not documented on Stripe.com
`console.log(Stripe)` prints a different list of functions than listed in the README.  There was a new version a couple days ago.  Maybe it's not updated here yet.

```
var Stripe = require('stripe')(config.stripePrivate);
Stripe.setApiVersion('2017-01-27');
```
(same same if I don't set the ApiVersion.  What version does it speak?)

To create a subscription for an existing customer:

  - `Stripe.subscriptions.create` doesn't exist in the module, but is [documented on Stripe.com](https://stripe.com/docs/api#create_subscription).
  - `Stripe.customerSubscriptions` is in the module, but not documented on Stripe.com.

Did I miss something?  Where is this API designed and documented?

Thanks!

Mike

```
{ _api:           
I20170131-20:06:03.848(-6)?    { auth: 'Basic opensesame',
I20170131-20:06:03.848(-6)?      host: 'api.stripe.com',
I20170131-20:06:03.848(-6)?      port: '443',
I20170131-20:06:03.849(-6)?      basePath: '/v1/',
I20170131-20:06:03.849(-6)?      version: '2017-01-27',
I20170131-20:06:03.849(-6)?      timeout: 120000,
I20170131-20:06:03.849(-6)?      agent: null,
I20170131-20:06:03.849(-6)?      dev: false },
I20170131-20:06:03.850(-6)?   account: 
I20170131-20:06:03.850(-6)?    { _stripe: [Circular],
I20170131-20:06:03.850(-6)?      _urlData: {},
I20170131-20:06:03.850(-6)?      basePath: [Function],
I20170131-20:06:03.851(-6)?      path: [Function] },
I20170131-20:06:03.851(-6)?   accounts: 
I20170131-20:06:03.851(-6)?    { _stripe: [Circular],
I20170131-20:06:03.851(-6)?      _urlData: {},
I20170131-20:06:03.851(-6)?      basePath: [Function],
I20170131-20:06:03.852(-6)?      path: [Function] },
I20170131-20:06:03.853(-6)?   applePayDomains: 
I20170131-20:06:03.854(-6)?    { _stripe: [Circular],
I20170131-20:06:03.854(-6)?      _urlData: {},
I20170131-20:06:03.854(-6)?      basePath: [Function],
I20170131-20:06:03.854(-6)?      path: [Function],
I20170131-20:06:03.854(-6)?      create: [Function],
I20170131-20:06:03.855(-6)?      list: [Function],
I20170131-20:06:03.855(-6)?      retrieve: [Function],
I20170131-20:06:03.855(-6)?      del: [Function] },
I20170131-20:06:03.855(-6)?   balance: 
I20170131-20:06:03.856(-6)?    { _stripe: [Circular],
I20170131-20:06:03.856(-6)?      _urlData: {},
I20170131-20:06:03.856(-6)?      basePath: [Function],
I20170131-20:06:03.856(-6)?      path: [Function] },
I20170131-20:06:03.857(-6)?   charges: 
I20170131-20:06:03.857(-6)?    { _stripe: [Circular],
I20170131-20:06:03.857(-6)?      _urlData: {},
I20170131-20:06:03.857(-6)?      basePath: [Function],
I20170131-20:06:03.857(-6)?      path: [Function],
I20170131-20:06:03.857(-6)?      create: [Function],
I20170131-20:06:03.857(-6)?      list: [Function],
I20170131-20:06:03.858(-6)?      retrieve: [Function],
I20170131-20:06:03.858(-6)?      update: [Function],
I20170131-20:06:03.858(-6)?      setMetadata: [Function],
I20170131-20:06:03.859(-6)?      getMetadata: [Function] },
I20170131-20:06:03.859(-6)?   countrySpecs: 
I20170131-20:06:03.860(-6)?    { _stripe: [Circular],
I20170131-20:06:03.860(-6)?      _urlData: {},
I20170131-20:06:03.860(-6)?      basePath: [Function],
I20170131-20:06:03.860(-6)?      path: [Function],
I20170131-20:06:03.861(-6)?      list: [Function],
I20170131-20:06:03.861(-6)?      retrieve: [Function] },
I20170131-20:06:03.861(-6)?   coupons: 
I20170131-20:06:03.861(-6)?    { _stripe: [Circular],
I20170131-20:06:03.862(-6)?      _urlData: {},
I20170131-20:06:03.862(-6)?      basePath: [Function],
I20170131-20:06:03.862(-6)?      path: [Function],
I20170131-20:06:03.862(-6)?      create: [Function],
I20170131-20:06:03.862(-6)?      list: [Function],
I20170131-20:06:03.863(-6)?      update: [Function],
I20170131-20:06:03.863(-6)?      retrieve: [Function],
I20170131-20:06:03.863(-6)?      del: [Function] },
I20170131-20:06:03.863(-6)?   customers: 
I20170131-20:06:03.863(-6)?    { _stripe: [Circular],
I20170131-20:06:03.864(-6)?      _urlData: {},
I20170131-20:06:03.864(-6)?      basePath: [Function],
I20170131-20:06:03.864(-6)?      path: [Function],
I20170131-20:06:03.864(-6)?      create: [Function],
I20170131-20:06:03.864(-6)?      list: [Function],
I20170131-20:06:03.864(-6)?      retrieve: [Function],
I20170131-20:06:03.864(-6)?      update: [Function],
I20170131-20:06:03.864(-6)?      del: [Function],
I20170131-20:06:03.865(-6)?      setMetadata: [Function],
I20170131-20:06:03.865(-6)?      getMetadata: [Function] },
I20170131-20:06:03.865(-6)?   disputes: 
I20170131-20:06:03.865(-6)?    { _stripe: [Circular],
I20170131-20:06:03.865(-6)?      _urlData: {},
I20170131-20:06:03.866(-6)?      basePath: [Function],
I20170131-20:06:03.866(-6)?      path: [Function],
I20170131-20:06:03.866(-6)?      list: [Function],
I20170131-20:06:03.866(-6)?      retrieve: [Function],
I20170131-20:06:03.866(-6)?      update: [Function],
I20170131-20:06:03.866(-6)?      setMetadata: [Function],
I20170131-20:06:03.866(-6)?      getMetadata: [Function] },
I20170131-20:06:03.867(-6)?   events: 
I20170131-20:06:03.867(-6)?    { _stripe: [Circular],
I20170131-20:06:03.867(-6)?      _urlData: {},
I20170131-20:06:03.867(-6)?      basePath: [Function],
I20170131-20:06:03.867(-6)?      path: [Function],
I20170131-20:06:03.867(-6)?      list: [Function],
I20170131-20:06:03.867(-6)?      retrieve: [Function] },
I20170131-20:06:03.868(-6)?   invoices: 
I20170131-20:06:03.868(-6)?    { _stripe: [Circular],
I20170131-20:06:03.868(-6)?      _urlData: {},
I20170131-20:06:03.868(-6)?      basePath: [Function],
I20170131-20:06:03.869(-6)?      path: [Function],
I20170131-20:06:03.869(-6)?      create: [Function],
I20170131-20:06:03.869(-6)?      list: [Function],
I20170131-20:06:03.869(-6)?      retrieve: [Function],
I20170131-20:06:03.869(-6)?      update: [Function] },
I20170131-20:06:03.869(-6)?   invoiceItems: 
I20170131-20:06:03.870(-6)?    { _stripe: [Circular],
I20170131-20:06:03.870(-6)?      _urlData: {},
I20170131-20:06:03.870(-6)?      basePath: [Function],
I20170131-20:06:03.870(-6)?      path: [Function],
I20170131-20:06:03.871(-6)?      create: [Function],
I20170131-20:06:03.871(-6)?      list: [Function],
I20170131-20:06:03.871(-6)?      retrieve: [Function],
I20170131-20:06:03.871(-6)?      update: [Function],
I20170131-20:06:03.871(-6)?      del: [Function],
I20170131-20:06:03.872(-6)?      setMetadata: [Function],
I20170131-20:06:03.872(-6)?      getMetadata: [Function] },
I20170131-20:06:03.872(-6)?   plans: 
I20170131-20:06:03.872(-6)?    { _stripe: [Circular],
I20170131-20:06:03.872(-6)?      _urlData: {},
I20170131-20:06:03.872(-6)?      basePath: [Function],
I20170131-20:06:03.873(-6)?      path: [Function],
I20170131-20:06:03.873(-6)?      create: [Function],
I20170131-20:06:03.873(-6)?      list: [Function],
I20170131-20:06:03.873(-6)?      retrieve: [Function],
I20170131-20:06:03.874(-6)?      update: [Function],
I20170131-20:06:03.874(-6)?      del: [Function] },
I20170131-20:06:03.874(-6)?   recipientCards: 
I20170131-20:06:03.874(-6)?    { _stripe: [Circular],
I20170131-20:06:03.874(-6)?      _urlData: {},
I20170131-20:06:03.875(-6)?      basePath: [Function],
I20170131-20:06:03.875(-6)?      path: [Function],
I20170131-20:06:03.875(-6)?      create: [Function],
I20170131-20:06:03.876(-6)?      list: [Function],
I20170131-20:06:03.876(-6)?      retrieve: [Function],
I20170131-20:06:03.876(-6)?      update: [Function],
I20170131-20:06:03.876(-6)?      del: [Function] },
I20170131-20:06:03.876(-6)?   recipients: 
I20170131-20:06:03.877(-6)?    { _stripe: [Circular],
I20170131-20:06:03.877(-6)?      _urlData: {},
I20170131-20:06:03.877(-6)?      basePath: [Function],
I20170131-20:06:03.877(-6)?      path: [Function],
I20170131-20:06:03.877(-6)?      create: [Function],
I20170131-20:06:03.877(-6)?      list: [Function],
I20170131-20:06:03.878(-6)?      retrieve: [Function],
I20170131-20:06:03.878(-6)?      update: [Function],
I20170131-20:06:03.878(-6)?      del: [Function],
I20170131-20:06:03.878(-6)?      setMetadata: [Function],
I20170131-20:06:03.878(-6)?      getMetadata: [Function] },
I20170131-20:06:03.878(-6)?   refunds: 
I20170131-20:06:03.879(-6)?    { _stripe: [Circular],
I20170131-20:06:03.879(-6)?      _urlData: {},
I20170131-20:06:03.880(-6)?      basePath: [Function],
I20170131-20:06:03.880(-6)?      path: [Function],
I20170131-20:06:03.880(-6)?      create: [Function],
I20170131-20:06:03.881(-6)?      list: [Function],
I20170131-20:06:03.882(-6)?      retrieve: [Function],
I20170131-20:06:03.882(-6)?      update: [Function] },
I20170131-20:06:03.882(-6)?   tokens: 
I20170131-20:06:03.882(-6)?    { _stripe: [Circular],
I20170131-20:06:03.882(-6)?      _urlData: {},
I20170131-20:06:03.883(-6)?      basePath: [Function],
I20170131-20:06:03.883(-6)?      path: [Function],
I20170131-20:06:03.883(-6)?      create: [Function],
I20170131-20:06:03.883(-6)?      retrieve: [Function] },
I20170131-20:06:03.883(-6)?   transfers: 
I20170131-20:06:03.884(-6)?    { _stripe: [Circular],
I20170131-20:06:03.884(-6)?      _urlData: {},
I20170131-20:06:03.884(-6)?      basePath: [Function],
I20170131-20:06:03.884(-6)?      path: [Function],
I20170131-20:06:03.884(-6)?      create: [Function],
I20170131-20:06:03.884(-6)?      list: [Function],
I20170131-20:06:03.885(-6)?      retrieve: [Function],
I20170131-20:06:03.885(-6)?      update: [Function],
I20170131-20:06:03.885(-6)?      setMetadata: [Function],
I20170131-20:06:03.885(-6)?      getMetadata: [Function] },
I20170131-20:06:03.885(-6)?   applicationFees: 
I20170131-20:06:03.886(-6)?    { _stripe: [Circular],
I20170131-20:06:03.886(-6)?      _urlData: {},
I20170131-20:06:03.886(-6)?      basePath: [Function],
I20170131-20:06:03.886(-6)?      path: [Function],
I20170131-20:06:03.886(-6)?      list: [Function],
I20170131-20:06:03.887(-6)?      retrieve: [Function] },
I20170131-20:06:03.887(-6)?   fileUploads: 
I20170131-20:06:03.887(-6)?    { _stripe: [Circular],
I20170131-20:06:03.887(-6)?      _urlData: {},
I20170131-20:06:03.887(-6)?      basePath: [Function],
I20170131-20:06:03.888(-6)?      path: [Function],
I20170131-20:06:03.888(-6)?      retrieve: [Function],
I20170131-20:06:03.889(-6)?      list: [Function] },
I20170131-20:06:03.890(-6)?   bitcoinReceivers: 
I20170131-20:06:03.890(-6)?    { _stripe: [Circular],
I20170131-20:06:03.890(-6)?      _urlData: {},
I20170131-20:06:03.890(-6)?      basePath: [Function],
I20170131-20:06:03.890(-6)?      path: [Function],
I20170131-20:06:03.891(-6)?      create: [Function],
I20170131-20:06:03.891(-6)?      list: [Function],
I20170131-20:06:03.891(-6)?      retrieve: [Function],
I20170131-20:06:03.892(-6)?      update: [Function],
I20170131-20:06:03.892(-6)?      setMetadata: [Function],
I20170131-20:06:03.892(-6)?      getMetadata: [Function] },
I20170131-20:06:03.892(-6)?   products: 
I20170131-20:06:03.892(-6)?    { _stripe: [Circular],
I20170131-20:06:03.892(-6)?      _urlData: {},
I20170131-20:06:03.893(-6)?      basePath: [Function],
I20170131-20:06:03.893(-6)?      path: [Function],
I20170131-20:06:03.893(-6)?      list: [Function],
I20170131-20:06:03.893(-6)?      retrieve: [Function],
I20170131-20:06:03.893(-6)?      update: [Function],
I20170131-20:06:03.893(-6)?      del: [Function] },
I20170131-20:06:03.894(-6)?   skus: 
I20170131-20:06:03.894(-6)?    { _stripe: [Circular],
I20170131-20:06:03.894(-6)?      _urlData: {},
I20170131-20:06:03.894(-6)?      basePath: [Function],
I20170131-20:06:03.894(-6)?      path: [Function],
I20170131-20:06:03.894(-6)?      list: [Function],
I20170131-20:06:03.894(-6)?      retrieve: [Function],
I20170131-20:06:03.895(-6)?      update: [Function],
I20170131-20:06:03.895(-6)?      del: [Function] },
I20170131-20:06:03.895(-6)?   orders: 
I20170131-20:06:03.895(-6)?    { _stripe: [Circular],
I20170131-20:06:03.895(-6)?      _urlData: {},
I20170131-20:06:03.896(-6)?      basePath: [Function],
I20170131-20:06:03.896(-6)?      path: [Function],
I20170131-20:06:03.896(-6)?      list: [Function],
I20170131-20:06:03.896(-6)?      retrieve: [Function],
I20170131-20:06:03.896(-6)?      update: [Function] },
I20170131-20:06:03.896(-6)?   orderReturns: 
I20170131-20:06:03.896(-6)?    { _stripe: [Circular],
I20170131-20:06:03.897(-6)?      _urlData: {},
I20170131-20:06:03.897(-6)?      basePath: [Function],
I20170131-20:06:03.897(-6)?      path: [Function],
I20170131-20:06:03.897(-6)?      list: [Function],
I20170131-20:06:03.897(-6)?      retrieve: [Function] },
I20170131-20:06:03.898(-6)?   subscriptions: 
I20170131-20:06:03.898(-6)?    { _stripe: [Circular],
I20170131-20:06:03.898(-6)?      _urlData: {},
I20170131-20:06:03.898(-6)?      basePath: [Function],
I20170131-20:06:03.898(-6)?      path: [Function],
I20170131-20:06:03.898(-6)?      list: [Function],
I20170131-20:06:03.899(-6)?      retrieve: [Function],
I20170131-20:06:03.899(-6)?      del: [Function] },
I20170131-20:06:03.899(-6)?   subscriptionItems: 
I20170131-20:06:03.899(-6)?    { _stripe: [Circular],
I20170131-20:06:03.899(-6)?      _urlData: {},
I20170131-20:06:03.899(-6)?      basePath: [Function],
I20170131-20:06:03.900(-6)?      path: [Function],
I20170131-20:06:03.900(-6)?      create: [Function],
I20170131-20:06:03.900(-6)?      list: [Function],
I20170131-20:06:03.900(-6)?      retrieve: [Function],
I20170131-20:06:03.900(-6)?      update: [Function],
I20170131-20:06:03.900(-6)?      del: [Function] },
I20170131-20:06:03.900(-6)?   threeDSecure: 
I20170131-20:06:03.901(-6)?    { _stripe: [Circular],
I20170131-20:06:03.901(-6)?      _urlData: {},
I20170131-20:06:03.901(-6)?      basePath: [Function],
I20170131-20:06:03.901(-6)?      path: [Function],
I20170131-20:06:03.901(-6)?      create: [Function],
I20170131-20:06:03.901(-6)?      retrieve: [Function] },
I20170131-20:06:03.901(-6)?   sources: 
I20170131-20:06:03.902(-6)?    { _stripe: [Circular],
I20170131-20:06:03.902(-6)?      _urlData: {},
I20170131-20:06:03.902(-6)?      basePath: [Function],
I20170131-20:06:03.902(-6)?      path: [Function],
I20170131-20:06:03.902(-6)?      create: [Function],
I20170131-20:06:03.902(-6)?      retrieve: [Function],
I20170131-20:06:03.902(-6)?      update: [Function],
I20170131-20:06:03.903(-6)?      setMetadata: [Function],
I20170131-20:06:03.903(-6)?      getMetadata: [Function] },
I20170131-20:06:03.903(-6)?   customerCards: 
I20170131-20:06:03.903(-6)?    { _stripe: [Circular],
I20170131-20:06:03.903(-6)?      _urlData: {},
I20170131-20:06:03.903(-6)?      basePath: [Function],
I20170131-20:06:03.904(-6)?      path: [Function],
I20170131-20:06:03.904(-6)?      create: [Function],
I20170131-20:06:03.904(-6)?      list: [Function],
I20170131-20:06:03.904(-6)?      retrieve: [Function],
I20170131-20:06:03.905(-6)?      update: [Function],
I20170131-20:06:03.905(-6)?      del: [Function] },
I20170131-20:06:03.905(-6)?   customerSubscriptions: 
I20170131-20:06:03.905(-6)?    { _stripe: [Circular],
I20170131-20:06:03.905(-6)?      _urlData: {},
I20170131-20:06:03.905(-6)?      basePath: [Function],
I20170131-20:06:03.906(-6)?      path: [Function],
I20170131-20:06:03.906(-6)?      create: [Function],
I20170131-20:06:03.906(-6)?      list: [Function],
I20170131-20:06:03.906(-6)?      retrieve: [Function],
I20170131-20:06:03.906(-6)?      update: [Function],
I20170131-20:06:03.907(-6)?      del: [Function] },
I20170131-20:06:03.907(-6)?   chargeRefunds: 
I20170131-20:06:03.907(-6)?    { _stripe: [Circular],
I20170131-20:06:03.907(-6)?      _urlData: {},
I20170131-20:06:03.907(-6)?      basePath: [Function],
I20170131-20:06:03.909(-6)?      path: [Function],
I20170131-20:06:03.909(-6)?      create: [Function],
I20170131-20:06:03.909(-6)?      list: [Function],
I20170131-20:06:03.909(-6)?      retrieve: [Function],
I20170131-20:06:03.910(-6)?      update: [Function] },
I20170131-20:06:03.910(-6)?   applicationFeeRefunds: 
I20170131-20:06:03.910(-6)?    { _stripe: [Circular],
I20170131-20:06:03.911(-6)?      _urlData: {},
I20170131-20:06:03.911(-6)?      basePath: [Function],
I20170131-20:06:03.911(-6)?      path: [Function],
I20170131-20:06:03.911(-6)?      create: [Function],
I20170131-20:06:03.911(-6)?      list: [Function],
I20170131-20:06:03.911(-6)?      retrieve: [Function],
I20170131-20:06:03.912(-6)?      update: [Function] },
I20170131-20:06:03.912(-6)?   transferReversals: 
I20170131-20:06:03.919(-6)?    { _stripe: [Circular],
I20170131-20:06:03.919(-6)?      _urlData: {},
I20170131-20:06:03.919(-6)?      basePath: [Function],
I20170131-20:06:03.920(-6)?      path: [Function],
I20170131-20:06:03.920(-6)?      create: [Function],
I20170131-20:06:03.920(-6)?      list: [Function],
I20170131-20:06:03.920(-6)?      retrieve: [Function],
I20170131-20:06:03.920(-6)?      update: [Function] } }
```

### PR DESCRIPTION
**Pull request by brandur**
Removes the long list of available Node APIs from the README, and
suggests instead that people refer to our canonical API reference.

This make a somewhat convenient quick reference, but unfortunately we
don't have any plausible way of keeping them up to date so they're
especially prone to becoming inaccurate as causing confusion as seen
in #297 and #298..

Fixes #297.

Fixes #298.

Thoughts @ob-stripe / @remi-stripe?

### CODE REVIEW

**Comment by ob-stripe** [comment]
:+1: Absolutely in favor of having a single source of truth.

File: README.md [hunk 1]
```diff
@@ -88,159 +93,12 @@ stripe.balance.retrieve({
...
+See the [Node API docs](https://stripe.com/docs/api/node#intro).
 
-## More information / wikis
+## More Information
 
- * **[In-depth Documentation](https://stripe.com/docs/api/node)**
- * [Version 2 Overview](https://github.com/stripe/stripe-node/wiki/Version-2)
  * [REST API Version](https://github.com/stripe/stripe-node/wiki/REST-API-Version)
  * [Error Handling](https://github.com/stripe/stripe-node/wiki/Error-Handling)
  * [Passing Options](https://github.com/stripe/stripe-node/wiki/Passing-Options)
```
Comment (remi-stripe) [inline README.md]: should we remove those too? The error handling should at least point to https://stripe.com/docs/api/node#errors instead

Comment (brandur) [inline README.md]: Yeah, good point. I took them out in 3db1037.

In my mind we have pretty good sources for this information elsewhere:

* Setting the API version: I added this to the configuration list in 29ff08995751605eaae337e0dba9a45ca8c31e90. It's also available in the reference docs.
* Error handling is better described in the API reference docs.
* Passing options are also addressed in the API reference docs.
* Using Connect is described just a few lines above in this README.

**Comment by brandur** [comment]
Thanks for taking a look!

**Pull request merged**